hkmc2/shared/src/test/mlscript/codegen/BadIdent.mls

```diff
@@ -0,0 +1,15 @@
+:global
```

Comment:
You no longer need `:global` for the commands block that starts on the first line of the file.

Reply:
Cool!